engine/frontend/src/component/ServiceInfo.js

```diff
@@ -30,32 +30,43 @@ const ServiceInfo = () => {
     const params = useParams()
     const {name: enclaveName, uuid: serviceUuid} = params;
 
-    useEffect(() => {
-        let stream;
-        const ctrl = new AbortController();
+    const startServiceLogCollector = (ctrl) => {
+        let stream
         const fetchLogs = async () => {
```

Comment:
very `nit`: might be worth calling this `fetchServiceLogs` to distinguish the `fetchLogs` from the enclave view